tests/ops/test_qubit_ops.py

```diff
@@ -821,6 +821,15 @@ def test_PauliRot_matrix_identity(
         )
 
         assert np.allclose(res, expected, atol=tol, rtol=0)
+        
+    def test_PauliRot_decomposition_Identity(self):
```

Comment:
Might be good to also add a test for the eigvals?

Reply:
Like for the identity case, or a more general one for all of `PauliRot`?

Reply:
Oh, I meant more so that neither of the tests seem to be explicitly calling `op.eigvals`

Reply:
Thanks, gotcha! Will add then.

Reply:
Another good call, as the eigenvalues for all-identity were not being computed correctly either! This is now fixed and tested.